### DISCUSSION TITLE
More granular TOC for autodoc-generated pages

### DISCUSSION THREAD

**Discussion by dgw** [pradyunsg/furo#468] [Questions] [answered]
The other recent thread about TOC got me thinking: The project I've switched to Furo for our next major release doesn't list much in the Contents sidebar for autodoc pages.

For example, there is no Contents sidebar for pages like https://unstable.docs.sopel.chat/package/tools/time.html that use `automodule`. I think it would be useful to have each method listed in the TOC sidebar, but don't see how to do that with options in Furo _or_ to have sphinx.ext.autodoc generate sections that Furo would then use as contents.

Perchance is this a very simple thing and I'm just searching for the wrong keywords in Furo's or autodoc's documentation? 😅

**Comment by pradyunsg** [comment, marked as answer]
See https://github.com/sphinx-doc/sphinx/issues/6316

This is currently a limitation of Sphinx's autodoc extension. Once Sphinx addresses that, it should fairly seamlessly work in Furo. :)

**Reply by dgw** [reply]
I see some alternatives to play with in that thread, too. Thanks for the pointer!
